local_storage_example/volumes/apigw.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     required:
       nodeSelectorTerms:
       - matchExpressions:
-        - key: dojot.components/group
+        - key: dojot.components/dojot
           operator: In
           values:
-          - dojot
+          - true
```

Comment:
Hi @marianoleonardo. I think the correct sintax is "true". But maybe "enabled" is better. What do you think about it?

Reply:
I totally agree! I'll make the modifications and tests and then request your review again.